### DISCUSSION TITLE
update a value in a single column using the API [May 03]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2011] [Discord Threads]
> 🙋‍♂️ **Shard**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### u can use bulk update  
#  
> 👨‍🚀 **wingkwong**  
#### with where clause specfied  
#  
> 🙋‍♂️ **Shard**  
#### How would a call like that look? or where can I see some examples?  
#  
> 👨‍🚀 **wingkwong**  
#### curl -X 'PATCH' \   
#### http://localhost:8080/api/v1/db/data/bulk/noco/sakila/testApi/all?where=(Id,gt,7) \   
#### -H "Content-Type: application/json" \   
#### -H 'accept: application/json' \   
#### -H 'xc-auth: <REDACTED>' \   
#### --data '{ "Title": "Updated" }'  
#  
> 👨‍🚀 **wingkwong**  
#### the url is `/api/v1/db/data/bulk/{orgs}/{projectName}/{tableName}/`  
#  
> 👨‍🚀 **wingkwong**  
#### u may create a dummy table n use swagger ui to try  
#  
> 🙋‍♂️ **Shard**  
#### Using that I get: `invalid input syntax for type integer: "all"`  
#  
> 🙋‍♂️ **Shard**  
#### nevermind, I figured it out 🙂  
#  
> 🙋‍♂️ **Shard**  
#### Thanks!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/970618616011718658)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
